packages/cli/src/commands/ui/create/angular.ts

```diff
@@ -136,7 +140,7 @@ export default class Angular extends Command {
     npm run start
 
     See package.json for other available commands.
-    Happy hacking !
+    Happy hacking!
```

Comment:
Check `packages\cli-e2e\__tests__\angular.specs.ts` L125 to update the regexp ;)